src/routes/github/github-router.ts

```diff
@@ -36,7 +48,7 @@ subRouter.use("/branch", csrfMiddleware, GithubBranchRouter);
 
 subRouter.use(jiraAdminPermissionsMiddleware); // This must stay after jiraSymmetricJwtMiddleware
 
-subRouter.use(GithubOAuthRouter);
+subRouter.use("/login",  GithubOAuthLoginGet);
```

Comment:
Are you sure we need this? I couldn't remember where we have a login link in the ui?

Reply:
No idea ;)